.circleci/config.yml

```diff
@@ -126,7 +129,8 @@ references:
             helm dep up ./codacy
             echo "Adding 'https://charts.codacy.com/${CHANNEL}'"
             helm repo add --username "${CHARTS_REPO_USER}" --password "${CHARTS_REPO_PASS}" codacy https://charts.codacy.com/${CHANNEL}
-            helm push ./codacy codacy
+            helm push ./codacy codacy --version "$(cat .version)"
```

Comment:
This replaces the `version`  in the "./codacy/Chart.yaml" file. You are already doing this with the ytool above. I don't think this change is needed.